app/models/user.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+module.exports = (sequelize, DataTypes) =>
+  sequelize.define(
+    'user',
+    {
+      id: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        unique: true,
+        autoIncrement: true,
+        primaryKey: true
+      },
```

Comment:
You don't need define this (in model definition), sequelize autogenerate id like this

Reply:
I tried not to define it and it takes the email as key:
![Screenshot from 2019-08-13 18-18-57](https://user-images.githubusercontent.com/3391647/62978339-6b859880-bdf7-11e9-84ee-d416751af6bd.png)

Reply:
I tried to not define id and it takes email as key:
![Screenshot from 2019-08-13 18-18-57](https://user-images.githubusercontent.com/3391647/62978519-d8992e00-bdf7-11e9-9424-0b28224124ff.png)
Is that what you meant?